iron-location.html

```diff
@@ -209,14 +209,17 @@
         // nothing to do, the URL didn't change
         return;
       }
+      // Need to use a full URL in case the containing page has a base URI.
+      var fullNewUrl =
+          window.location.protocol + '//' + window.location.host + newUrl;
```

Comment:
+1, I was worried that this wasn't covered by our URL polyfill but it looks like it's covered. Basic testing confirmed